docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+---
+sidebar_label: jira
+description: Integrates with Jira to manage issues, comments, and track promotion workflows.
+---
+
+# `jira`
+
+:::info Enterprise Feature
+This extension is only available in Akuity Promote, you can use this feature in Kargo v1.6+ offered on [Akuity Platform](https://akuity.io/akuity-platform).
+:::
+
+The `jira` extension provides comprehensive integration with Jira, allowing you to create, update, delete, and search for issues, manage comments, and track promotion workflows. This extension is particularly useful for maintaining traceability between your deployment processes and project management activities.
```

Comment:
With tables and long URLs being notable exceptions, we try to observe a (soft) limit of 80 chars per line. It may seem silly, but it works out a lot better when reviewing PRs from a mobile device or viewing many docs or pieces of code side-by-side on a larger screen.

Reply:
Another matter of terminology, I think we could afford to be less generic than "extension" and say "This _promotion step_..."

imho, this is more straightforward for the reader.

Applies in a few other spots throughout this doc.